### PR TITLE
Minor documentation updates and id extraction in part 4 now also facilitates idloc 3, 6, and 7

### LINKED ISSUE #528: do.brondcounts not documented in vignette and params objects
just noticed that input argument do.brondcounts is not documented in vignette and params object.

### LINKED ISSUE #529: Part4 not able to work with idloc other than idloc = 2 and idloc = 5
**Describe the bug**

Message in google group:

> Hello,
> I am having trouble getting GGIR to recognize IDs consistently, which is leading to problems with part 4 and integrating sleep logs. I am using Axivity OMGUI to collect data which outputs files in the format deviceID_subjectID.cwa (example, 67067_0000000131.cwa). Initially I used idloc=6, which should give “67067_0000000131” as the ID. This works for part 2 and part 5 output, which have “67067_0000000131” in the ID column.
> 
> However, for part 4 output, the ID column is “67067,” and for the summary csv, all subjects using this device ID have been collapsed into the same subject. Further, part 4 is only able to match sleep log data when I use “67067” in the ID column, which is problematic because this device ID isn’t specific to the subject. Is there a way to get around this issue?
> 
> I'd appreciate any help, thanks very much!
> Joe 

**Proposed solution:**

Revise https://github.com/wadpac/GGIR/blob/master/R/g.part4_extractid.R such that it can handle more idloc variants. Currently it is limited to idloc = 2 and idloc = 5.

### PR DESCRIPTION
**Pull request by vincentvanhees**
fixes #528 -> minor documentation updates
fixes #529 -> id extraction in part 4 now also facilitates idloc 3, 6, and 7

Checklist before merging:

- [x] Existing tests still work (check by running the test suite, e.g. from RStudio).
- [ ] Added tests (if you added functionality) or fixed existing test (if you fixed a bug).
- [x] Updated or expanded the documentation.
- [x] Updated release notes in `inst/NEWS.Rd` with a user-readable summary. Please, include references to relevant issues or PR discussions.
- [ ] Added your name to the contributors lists in the `DESCRIPTION` and `CITATION.cff` files.

### CODE REVIEW

**Comment by vincentvanhees** [comment]
Hi @jhmigueles this is small pull request with minor changes. If you have time can you have a quick look?

**Comment by jhmigueles** [comment]
A couple of things about the id extraction:
- In the pdf manual it says that "You may have noticed that idloc 3 and 4 are skipped, they were used for one study in 2012, and not actively maintained anymore, but because it is legacy code not omitted". I can see that idloc = 3 is handled in the code but idloc = 4 is not. Is this ok? Should we document what GGIR does if idloc = 3 is specified?
- Would it be a good idea to let the user specify the character that separates the id from the rest of the filename? I think this would be more intuitive, something like **idloc = "_"** 

I can work on this tomorrrow if you think it's a good idea. Otherwise, the code looks good as it is.

File: man/g.part1.Rd [hunk 1]
```diff
@@ -148,7 +148,7 @@
         package activityCounts. We call them BrondCounts because
         there are large number of acitivty counts in the physical activity and
         sleep research field. By calling them Brond Counts we clarify that 
-        these are the counts proposed by Jan Brond and implemented in R by
+        these are the counts proposed by Jan Br&#248;nd and implemented in R by
         Ruben Brondeel. The Brond Counts are intended to be an imitation of
```
Comment (jhmigueles) [inline man/g.part1.Rd:152]: Should we revise and replace Brond for Br&#248;nd everywhere? Here it also appears in line 152.

**Comment by vincentvanhees** [comment]
thanks Jairo
> In the pdf manual it says that "You may have noticed that idloc 3 and 4 are skipped, they were used for one study in 2012, and not actively maintained anymore, but because it is legacy code not omitted". I can see that idloc = 3 is handled in the code but idloc = 4 is not. Is this ok? Should we document what GGIR does if idloc = 3 is specified?

idloc = 3 was implemented for the Pelotas study (study around which the GGIR development started). However, I now realise that it is effectively the same as idloc = 5, so we can omit idloc3 and keep the documentation as it is. I will do that later today.

> Would it be a good idea to let the user specify the character that separates the id from the rest of the filename? I think this would be more intuitive, something like idloc = "_"

Although I agree that this is more userfriendly, I think we should keep it as it is for the following reasons:
- We would have to facilitate both the old and the new format to keep R scripts from users compatible and avoid that the code gets more complex.
- To keep idloc flexible to any location of the id value and not only to indicate where the ID is before a certain character in the file name. For example, I think Axivity puts ID towards the end of the filename and I have never made a good effort to also design an idloc option for that.

I agree that it may be good to think how this can be made more user-friendly, but let's not rush it as I am a bit worried that it is a non-trivial task.

File: man/g.part1.Rd (hunk 1 above)
Comment (vincentvanhees) [reply man/g.part1.Rd:152]: Good point, thanks.

**Comment by vincentvanhees** [comment]
I have now made a couple of edits:

- We spell brondcounts when we refer to the metric only (e.g. in argument `do.brondcounts`). This is without the &#248;, because it also refers to Brondeel and having a metric name with a special symbols is not practical.
- We spell Br&#248;nd when specifically referring to Jan Br&#248;nd in the documentation when crediting his work.
- I am no longer refer to brondcounts as Brond Counts, because that would be inconsistent with the argument name and may be confusing.
- `idloc` handling in part 4 is now simplified to: First attempt to extract ID from part 3 milestone data where it was already extracted. If this fails (not sure when, but for now I am keeping it) then attempt to extract ID from filename again based on idloc 2, 5, 6 or 7. In this way we try to avoid duplicated extraction of ID.

**Pull request merged**